Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -632,19 +632,33 @@ public void HandleActionPutItemInContainer(uint itemGuid, uint containerGuid, in
                         return;
                     }
 
-                    var returnStance = new Motion(CurrentMotionState.Stance);
-
                     if (!success)
                     {
                         Session.Network.EnqueueSend(new GameEventInventoryServerSaveFailed(Session, itemGuid, WeenieError.ActionCancelled));
-                        EnqueueBroadcastMotion(returnStance);
+                        return;
+                    }
+
+                    // Was this item picked up by someone else?
+                    if (itemRootOwner == null && item.CurrentLandblock == null)
+                    {
+                        Session.Network.EnqueueSend(new GameEventInventoryServerSaveFailed(Session, itemGuid, WeenieError.ActionCancelled));
```

Comment:
this should log the exact error info that was posted

logging dupe attempts is a feature that the admins have found very useful so far. even if the attempt was unsuccessful, it allows them to monitor people who are trying to exploit